WORKSPACE

```diff
@@ -131,6 +146,7 @@ maven_install(
         "androidx.navigation:navigation-ui:2.0.0",
         "androidx.navigation:navigation-ui-ktx:2.0.0",
         "androidx.recyclerview:recyclerview:1.0.0",
+        "androidx.room:room-runtime:2.2.5",
```

Comment:
NB that some of these two new deps probably aren't needed, but we aren't shipping the Bazel version of the app.